src/wasm-validator.h

```diff
@@ -27,14 +27,15 @@
 namespace wasm {
 
 struct WasmValidator : public PostWalker<WasmValidator, Visitor<WasmValidator>> {
-  bool valid;
+  bool valid = true;
+  bool validateWeb = false;
```

Comment:
these have similar names but very different meanings. how about the longer name used below for the second, `validateWebConstraints`?